spec_helper.rb

```diff
@@ -48,10 +48,7 @@
   end
 end
 
-dir = "../fixtures/code"
-CODE_LOADING_DIR = defined?(require_relative) ?
-                     File.realpath(dir, __FILE__) :
-                     File.expand_path(dir, __FILE__)
+CODE_LOADING_DIR = File.expand_path "../fixtures/code", __FILE__
```

Comment:
@elia actually if I remember correctly the issue was not that we are missing `realpath` (it's implemented in nodejs), but it was returning a path that was missing one (the first) component. I tried to figure it out and gave up. I think it's something to do with how nodejs sees current working directory. I gave up mostly because I didn't see the point of that code in the first place. @eregon, pardon my ignorance, what's the rationale for branching on `defined?(require_relative)` here? I.e. what problem does that solve as opposed to simply using `File.expand_path "../fixtures/code", __FILE__` unconditionally?